components/eventsrv-protocol/build.rs

```diff
@@ -26,7 +26,6 @@ fn generate_protocols() {
     let cmd = Command::new(format!("{}/bin/protoc", prefix))
         .arg("--rust_out")
         .arg(out_dir)
-        .arg("--doc_out=markdown,index.md:doc")
```

Comment:
I removed this because it was raising errors and none of the other protocol generating crates had it.